includes/sdk-device-id/transfer-to-a-new-device.md

```diff
@@ -0,0 +1,3 @@
+#### Transfer to a new device
+
+When a user switch to a new device and transfers data to the new device, it's possible that the device ID is preserved and two different devices can have the same device ID. The specifics of what is backed up can vary from app to app. In general, most apps will have their data backed up, including any databases and file directories that are part of the app. However, it ultimately depends on how the app has been designed and what the developers have chosen to include in the backup. If the database or file directories has been backed up from one device to another since the device ID has been stored there and the SDK tries to fetch it during the initialization, it’s possible that different devices use the same device ID.
```

Comment:
Can you please reword this? I'm not sure I understand. Is this about multiple devices potentially having the same device id on accident?

Some thing I expected here:
* How Amplitude ID works to connect the same user across multiple device id's
* How to pass Device Id from one instance of the SDK to another

Reply:
Yes, this is a possible cause of using mobile SDKs and having the same device ID across different devices. 
This is for [a discourse question](https://discourse.amplitude.com/t/same-device-id-in-different-devices/10897/7) 

> How Amplitude ID works to connect the same user across multiple device id's

Add a separate section and say that Amplitude merges different device IDs into the same Amplitude ID if user ID is the same 

> How to pass Device Id from one instance of the SDK to another

I have that for browser JS SDK which is frequently asked and make up missing get and set device id. But I don't see any user has multiple instances installed on mobile devices. Maybe we could add full example when it is asked?